### PR TITLE
[hi] Localize cloud native apps.md to Hindi

### LINKED ISSUE #1536: [New term] `cloud native apps` for `Hindi`
### Please describe why this term needs to be updated

What is this term about? (briefly)

- Cloud native applications are specifically designed to take advantage of innovations in cloud computing.

Why does this term need to be added? (briefly)

- Localizing this term in Hindi will help Hindi folks learn about this term in their native language.

Link here: 

- https://glossary.cncf.io/cloud-native-apps/

### PR DESCRIPTION
**Pull request by Martande8055**
Signed-off-by: Saurabh Subhash Martande <smartande8055@gmail.com>

### Describe your changes
localize cloud native apps.md to Hindi.

### Related issue number or link (ex: `resolves #issue-number`)
 resolves #1536 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Martande8055** [comment]
Is there any update on this PR?

File: content/hi/cloud native apps.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+प्रत्येक डेटासेंटर के पास विशिष्ट परिवेशों के लिए [कसकर युग्मित](/tightly-coupled-architectures/) अनुप्रयोग वाली सेवाएँ थीं,
+[वर्चुअल मशीन](/virtual-machine/) और सेवाओं जैसी बुनियादी सुविधाओं के लिए अक्सर मैन्युअल प्रावधान पर बहुत अधिक भरोसा करते हैं।
+यह, बदले में, उस विशिष्ट डेटासेंटर के लिए डेवलपर्स और उनके अनुप्रयोगों को विवश करता है।
+क्लाउड के लिए डिज़ाइन नहीं किए गए एप्लिकेशन क्लाउड वातावरण की लचीलापन और स्केलिंग क्षमताओं का लाभ नहीं उठा सकते।
+उदाहरण के लिए, जिन ऐप्स को सही ढंग से शुरू करने के लिए मैन्युअल हस्तक्षेप की आवश्यकता होती है, वे स्वचालित रूप से स्केल नहीं कर सकते हैं,
+नही विफलता की स्थिति में उन्हें स्वचालित रूप से पुनरारंभ किया जा सकता है।
+
+## समाधान
+
+जबकि क्लाउड नेटिव एप्लिकेशन के लिए कोई "एक आकार सभी फिट बैठता है" पथ नहीं है, उनमें कुछ समानताएँ हैं।
```
Comment (shivam-sharma7) [inline content/hi/cloud native apps.md:28]:
```suggestion
जबकि क्लाउड नेटिव एप्लिकेशन के लिए कोई एक आकार सभी फिट बैठता है। पाथ नहीं है, उनमें कुछ समानताएँ हैं।
```

Comment (shivam-sharma7) [reply content/hi/cloud native apps.md:28]: Need to add suggested changes

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]

File: content/hi/cloud native apps.md (hunk 1 above)
Comment (Martande8055) [reply content/hi/cloud native apps.md]: changes Done

File: content/hi/cloud native apps.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ये एप्लिकेशन अपने संबंधित क्लाउड आर्किटेक्चर के साथ आसानी से एकीकृत होते हैं,
+क्लाउड के संसाधनों और [स्केलिंग](/scalability/) क्षमताओं का लाभ उठाना,
+यह उन अनुप्रयोगों को भी संदर्भित करता है जो क्लाउड कंप्यूटिंग द्वारा संचालित अवसंरचना में नवाचारों का लाभ उठाते हैं।
+क्लाउड नेटिव एप्लिकेशन में आज ऐसे ऐप्स शामिल हैं जो क्लाउड प्रदाता के डेटासेंटर में और क्लाउड नेटिव प्लेटफॉर्म ऑन-प्रिमाइसेस में चलते हैं।
+
+## समस्या
+
+परंपरागत रूप से, ऑन-प्रिमाइसेस परिवेशों ने काफी पहले से आरक्षित तरीके से कंप्यूट संसाधन प्रदान किए।
+प्रत्येक डेटासेंटर के पास विशिष्ट परिवेशों के लिए [कसकर युग्मित](/tightly-coupled-architectures/) अनुप्रयोग वाली सेवाएँ थीं,
+[वर्चुअल मशीन](/virtual-machine/) और सेवाओं जैसी बुनियादी सुविधाओं के लिए अक्सर मैन्युअल प्रावधान पर बहुत अधिक भरोसा करते हैं।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:20]:
```suggestion
[वर्चुअल मशीन](/virtual-machine/) और अन्य सेवाऍ जैसी बुनियादी सुविधाओं के लिए अक्सर मैन्युअल प्रावधान पर बहुत अधिक भरोसा किया जाता है।
```

File: content/hi/cloud native apps.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+यह, बदले में, उस विशिष्ट डेटासेंटर के लिए डेवलपर्स और उनके अनुप्रयोगों को विवश करता है।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:21]:
```suggestion
यह बदले में उस विशिष्ट डेटासेंटर के लिए डेवलपर्स और उनके अनुप्रयोगों को विवश करता है।
```

File: content/hi/cloud native apps.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+क्लाउड के लिए डिज़ाइन नहीं किए गए एप्लिकेशन क्लाउड वातावरण की लचीलापन और स्केलिंग क्षमताओं का लाभ नहीं उठा सकते।
+उदाहरण के लिए, जिन ऐप्स को सही ढंग से शुरू करने के लिए मैन्युअल हस्तक्षेप की आवश्यकता होती है, वे स्वचालित रूप से स्केल नहीं कर सकते हैं,
+नही विफलता की स्थिति में उन्हें स्वचालित रूप से पुनरारंभ किया जा सकता है।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:24]:
```suggestion
ना ही विफलता की स्थिति में उन्हें स्वचालित रूप से पुनरारंभ किया जा सकता है।
```

File: content/hi/cloud native apps.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+यह उन अनुप्रयोगों को भी संदर्भित करता है जो क्लाउड कंप्यूटिंग द्वारा संचालित अवसंरचना में नवाचारों का लाभ उठाते हैं।
+क्लाउड नेटिव एप्लिकेशन में आज ऐसे ऐप्स शामिल हैं जो क्लाउड प्रदाता के डेटासेंटर में और क्लाउड नेटिव प्लेटफॉर्म ऑन-प्रिमाइसेस में चलते हैं।
+
+## समस्या
+
+परंपरागत रूप से, ऑन-प्रिमाइसेस परिवेशों ने काफी पहले से आरक्षित तरीके से कंप्यूट संसाधन प्रदान किए।
+प्रत्येक डेटासेंटर के पास विशिष्ट परिवेशों के लिए [कसकर युग्मित](/tightly-coupled-architectures/) अनुप्रयोग वाली सेवाएँ थीं,
+[वर्चुअल मशीन](/virtual-machine/) और सेवाओं जैसी बुनियादी सुविधाओं के लिए अक्सर मैन्युअल प्रावधान पर बहुत अधिक भरोसा करते हैं।
+यह, बदले में, उस विशिष्ट डेटासेंटर के लिए डेवलपर्स और उनके अनुप्रयोगों को विवश करता है।
+क्लाउड के लिए डिज़ाइन नहीं किए गए एप्लिकेशन क्लाउड वातावरण की लचीलापन और स्केलिंग क्षमताओं का लाभ नहीं उठा सकते।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:22]:
```suggestion
क्लाउड के लिए डिज़ाइन नहीं किए गए एप्लिकेशन क्लाउड वातावरण की लचीलापन (resilience) और स्केलिंग क्षमताओं का लाभ नहीं उठा सकते।
```

File: content/hi/cloud native apps.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+उदाहरण के लिए, जिन ऐप्स को सही ढंग से शुरू करने के लिए मैन्युअल हस्तक्षेप की आवश्यकता होती है, वे स्वचालित रूप से स्केल नहीं कर सकते हैं,
+नही विफलता की स्थिति में उन्हें स्वचालित रूप से पुनरारंभ किया जा सकता है।
+
+## समाधान
+
+जबकि क्लाउड नेटिव एप्लिकेशन के लिए कोई एक आकार सभी फिट बैठता है। पाथ नहीं है, उनमें कुछ समानताएँ हैं।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:28]:
```suggestion
जबकि क्लाउड नेटिव एप्लिकेशन के लिए कोई एक आकार तभी फिट बैठता है, जब उनमें कुछ समानताएँ हों।
```

File: content/hi/cloud native apps.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+क्लाउड नेटिव ऐप्स लचीला, प्रबंधनीय और उनके साथ आने वाली क्लाउड सेवाओं के सूट द्वारा सहायता प्राप्त हैं।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:29]:
```suggestion
क्लाउड नेटिव ऐप्स लचीले व प्रबंधनीय होते हैं और उनके साथ आने वाली क्लाउड सेवाओं द्वारा उन्हें सहायता प्राप्त होती है।
```

File: content/hi/cloud native apps.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+विभिन्न क्लाउड सेवाएं उच्च स्तर की [अवलोकन क्षमता](/observability/) को सक्षम बनाती हैं,
+इससे पहले कि वे आगे बढ़ें, उपयोगकर्ताओं को समस्याओं का पता लगाने और उनका समाधान करने में सक्षम बनाता है।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:31]:
```suggestion
इससे पहले कि वे आगे बढ़ें, उपयोगकर्ताओं को समस्याओं का पता लगाने और उनका समाधान करने में सक्षम बनाती है।
```

File: content/hi/cloud native apps.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+मजबूत स्वचालन के साथ संयुक्त, वे इंजीनियरों को न्यूनतम परिश्रम के साथ बार-बार और अनुमानित रूप से उच्च प्रभाव वाले परिवर्तन करने की अनुमति देते हैं।
```
Comment (aj11anuj) [inline content/hi/cloud native apps.md:32]:
```suggestion
मजबूत स्वचालन के साथ संयुक्त, वे इंजीनियरों को न्यूनतम परिश्रम के साथ बार-बार और अनुमानित रूप से उच्च प्रभाव वाले परिवर्तन करने की अनुमति देती हैं।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]

**Comment by abhay-raj19** [comment]
Looks good to me @Martande8055 .
Thanks for contributing.

File: content/hi/cloud native apps.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+क्लाउड के लिए डिज़ाइन नहीं किए गए एप्लिकेशन क्लाउड वातावरण की लचीलापन (resilience) और स्केलिंग क्षमताओं का लाभ नहीं उठा सकते।
+उदाहरण के लिए, जिन ऐप्स को सही ढंग से शुरू करने के लिए मैन्युअल हस्तक्षेप की आवश्यकता होती है, वे स्वचालित रूप से स्केल नहीं कर सकते हैं,
+ना ही विफलता की स्थिति में उन्हें स्वचालित रूप से पुनरारंभ किया जा सकता है।
+
+## समाधान
+
+जबकि क्लाउड नेटिव एप्लिकेशन के लिए कोई एक आकार तभी फिट बैठता है, जब उनमें कुछ समानताएँ हों।
+क्लाउड नेटिव ऐप्स लचीले व प्रबंधनीय होते हैं और उनके साथ आने वाली क्लाउड सेवाओं द्वारा उन्हें सहायता प्राप्त होती है।
+विभिन्न क्लाउड सेवाएं उच्च स्तर की [अवलोकन क्षमता](/observability/) को सक्षम बनाती हैं,
+इससे पहले कि वे आगे बढ़ें, उपयोगकर्ताओं को समस्याओं का पता लगाने और उनका समाधान करने में सक्षम बनाती है।
```
Comment (bishal7679) [inline content/hi/cloud native apps.md:31]:
```suggestion
विभिन्न क्लाउड सेवाएँ उपयोगकर्ताओं को समस्याओं के बढ़ने से पहले उनका पता लगाना और उनका समाधान करने में सक्षम बनाने के साथ साथ [अवलोकन योग्यता](/observability/) की एक उच्च स्तर को सक्षम करती हैं।
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]
Left some suggestions!

**Review by bishal7679 — APPROVED** [review]

**Pull request merged**